pkgs/default.nix

```diff
@@ -10,6 +10,6 @@
   pylightning = pkgs.python3Packages.callPackage ./pylightning { };
   liquid-swap = pkgs.python3Packages.callPackage ./liquid-swap { };
   generate-secrets = pkgs.callPackage ./generate-secrets { };
-
   pinned = import ./pinned.nix;
+  nixops19_09 = pkgs.callPackage ./nixops { };
```

Comment:
Nit: I'd keep that visual separation:  Normal packages, blank line, pinned.

Reply:
Sure, fixed